pom.xml

```diff
@@ -121,6 +121,24 @@
       <scope>test</scope>
     </dependency>
 
+	  <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+	  </dependency>
+
+	  <dependency>
+      <groupId>net.sf.m-m-m</groupId>
+      <artifactId>mmm-util-core</artifactId>
+      <version>8.7.0</version>
+	  </dependency>
+
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
+      <version>5.1.9.RELEASE</version>
+    </dependency>
```

Comment:
Please follow code-style and fix indendation :)

Reply:
This is very strange... In my IDE they are indented just fine. I don't know why it looks so messed up in GitHub.

Reply:
This is comming from a mixture of Tabs and Spaces. That is also why `devonfw` discourages to use Tabs. Actually Tabs are from the IT stoneage century where people had to optimize single characters to store the data on a punch card or floppy disc and therefore used one tab instead of 4 spaces to "compress" texts. Then different OS and editors had different interpretations how many spaces a tab should be. Nowadays this can be configured but there is no standard. To make it short: Tabs suck and indendation should always be done with whitespaces. The number of whitespaces per indendation is also not globally agreed. Important is to decide for some value and then stick to it. `devonfw` decided for 2 spaces what seems a good compromise. After all it is a matter of taste but there is no point and value in discussions whether 2, 3, or 4 spaces are better. Just stick to one choice and do not bother anymore...

Reply:
You might want to configure your git GUI tool to show tabs differently (e.g. as "-->" instead of "  " or "     ") so you will more easily see what is going on.
In Eclipse you can also configure `General > Editors > Text Editors > Show whitespace characters`.